### PR TITLE
TOOLSDOC-718: Added CDK tooling how to

### PR DESCRIPTION
**Pull request by mishaone**
@xcoulon - could you review this please. I still have some comments from Marian to implement, but I'll do that tomorrow on the google doc and update this with minor updates once he signs off on those changes.

### CODE REVIEW

**Comment by xcoulon** [comment]
@mishaone this looks good to me. @fbricon and @alexeykazakov, would you mind reviewing the CDK installer and OpenShift parts ? Also, what about linking the release blog post to this how-to ?

**Comment by mishaone** [comment]
@xcoulon they've reviewed the google doc yesterday and I'm still working out some minor issues there with Marian tomorrow.

**Comment by mishaone** [comment]
I left this far longer than I should have, but @xcoulon this is the article you reviewed yesterday. Can you let me know if this is all good and ready to tag public for GA later?

File: documentation/howto/cdk_howto.adoc [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+.. Navigate to the **Images** node under the Docker connection.
+.. Right-click your image and click **Run**.
+.. Fill in the necessary details and click **Finish** to run your image.  It is optional to give the container a name, but it is recommended to name it. This name helps locate the specific container in a list of containers in the future.
+. In the **Docker Explorer** view, select the container and expand its node and select the **8080** port and click on **Show In**>**Web browser** to access the application deployed in the Docker container.
+
+=== Next Steps for the Docker Tooling
+
+For further information about the basics of Docker Tooling, see link:docker_basics.html[Configure Docker Tooling (Basic)]
+
+=== Using OpenShift Tooling
```
Comment (xcoulon) [inline documentation/howto/cdk_howto.adoc]: I would add a level 2 section title here, such as: "OpenShift Tooling" to better distinguish with the Docker tooling described above.

File: documentation/howto/index.adoc [hunk 2]
```diff
@@ -15,6 +15,8 @@ link:configure_remote_server.html[Configure a Remote Server]
 
 link:eapwfk_configure_ide.html[Configure JBoss EAP and JBoss Web Framework Kit for the IDE]
 
+link:cdk_howto.html[Using the IDE's Container Development Kit Tooling]
+
```
Comment (xcoulon) [inline documentation/howto/index.adoc]: Could we had the `CDK` acronym at the end of the link title, so user can search for 'CDK' in the page ?

File: documentation/howto/cdk_howto.adoc [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+.. In the **OpenShift Explorer** view, right-click the appropriate service under the relevant OpenShift Project.
+.. In the **Select a server type** field, type **openshift**, and from the listed results, select **OpenShift 3 Server Adapter** and click **Next**.
+.. In the displayed context menu, select **Server Adapter**. This displays a New Server wizard which has the adapter type and connection already selected.
+.. Edit any of the settings as required, and then click **Finish**.
+. Debug the application, if required.
+.. In the **Server** view, from the context of a server adapter, select **Restart in Debug** to debug the application.
+
+=== Next Steps for the OpenShift Tooling
+
+See the **Developing for the Cloud with OpenShift 3** section in the index for additional tasks using the OpenShift 3 tooling.
```
Comment (xcoulon) [inline documentation/howto/cdk_howto.adoc]: This should be a link

**Comment by xcoulon** [comment]
Changes applied, then PR merged.

**Pull request closed without merging**